consensus/state.go

```diff
@@ -1474,7 +1483,9 @@ func (cs *ConsensusState) tryAddVote(vote *types.Vote, peerID p2p.ID) (bool, err
 		if err == ErrVoteHeightMismatch {
 			return added, err
 		} else if voteErr, ok := err.(*types.ErrVoteConflictingVotes); ok {
-			if bytes.Equal(vote.ValidatorAddress, cs.privValidator.GetAddress()) {
+			addr, err := cs.privValidator.GetAddress()
+			cs.Logger.Error("Can not retrieve privValidator's address", "err", err)
```

Comment:
where's if?

Reply:
```suggestion
			if err != nil {
				cs.Logger.Error("Failed to get private validator address", "err", err)
				return
			}

Reply:
Thanks for catching this! (must be `return added, err` though)